Software/src/LedDeviceManager.cpp

```diff
@@ -571,31 +573,34 @@ void LedDeviceManager::disconnectSignalSlotsLedDevice()
 		qWarning() << Q_FUNC_INFO << "m_ledDevice == NULL";
 		return;
 	}
-
-	disconnect(m_ledDevice, SIGNAL(commandCompleted(bool)),		this, SLOT(ledDeviceCommandCompleted(bool)));
-	disconnect(m_ledDevice, SIGNAL(ioDeviceSuccess(bool)),		this, SLOT(ledDeviceIoDeviceSuccess(bool)));
-	disconnect(m_ledDevice, SIGNAL(openDeviceSuccess(bool)),	this, SLOT(ledDeviceOpenDeviceSuccess(bool)));
-
-	disconnect(m_ledDevice, SIGNAL(firmwareVersion(QString)),		this, SIGNAL(firmwareVersion(QString)));
-	disconnect(m_ledDevice, SIGNAL(firmwareVersionUnofficial(int)), this, SIGNAL(firmwareVersionUnofficial(int)));
-	disconnect(m_ledDevice, SIGNAL(colorsUpdated(QList<QRgb>)),		this, SIGNAL(setColors_VirtualDeviceCallback(QList<QRgb>)));
-
-	disconnect(this, SIGNAL(ledDeviceOpen()),								m_ledDevice, SLOT(open()));
-	disconnect(this, SIGNAL(ledDeviceSetColors(QList<QRgb>)),				m_ledDevice, SLOT(setColors(QList<QRgb>)));
-	disconnect(this, SIGNAL(ledDeviceOffLeds()),							m_ledDevice, SLOT(switchOffLeds()));
-	disconnect(this, SIGNAL(ledDeviceSetUsbPowerLedDisabled(bool)),			m_ledDevice, SLOT(setUsbPowerLedDisabled(bool)));
-	disconnect(this, SIGNAL(ledDeviceSetRefreshDelay(int)),					m_ledDevice, SLOT(setRefreshDelay(int)));
-	disconnect(this, SIGNAL(ledDeviceSetColorDepth(int)),					m_ledDevice, SLOT(setColorDepth(int)));
-	disconnect(this, SIGNAL(ledDeviceSetSmoothSlowdown(int)),				m_ledDevice, SLOT(setSmoothSlowdown(int)));
-	disconnect(this, SIGNAL(ledDeviceSetGamma(double, bool)),				m_ledDevice, SLOT(setGamma(double, bool)));
-	disconnect(this, SIGNAL(ledDeviceSetBrightness(int, bool)),				m_ledDevice, SLOT(setBrightness(int, bool)));
-	disconnect(this, SIGNAL(ledDeviceSetBrightnessCap(int, bool)),			m_ledDevice, SLOT(setBrightnessCap(int, bool)));
-	disconnect(this, SIGNAL(ledDeviceSetColorSequence(QString)),			m_ledDevice, SLOT(setColorSequence(QString)));
-	disconnect(this, SIGNAL(ledDeviceSetLuminosityThreshold(int, bool)),	m_ledDevice, SLOT(setLuminosityThreshold(int, bool)));
-	disconnect(this, SIGNAL(ledDeviceSetMinimumLuminosityEnabled(bool, bool)),	m_ledDevice, SLOT(setMinimumLuminosityThresholdEnabled(bool, bool)));
-	disconnect(this, SIGNAL(ledDeviceRequestFirmwareVersion()),				m_ledDevice, SLOT(requestFirmwareVersion()));
-	disconnect(this, SIGNAL(ledDeviceUpdateWBAdjustments()),				m_ledDevice, SLOT(updateWBAdjustments()));
-	disconnect(this, SIGNAL(ledDeviceUpdateDeviceSettings()),				m_ledDevice, SLOT(updateDeviceSettings()));
+	m_ledDevice->disconnect(this, nullptr, nullptr, nullptr);
+	disconnect(m_ledDevice, nullptr, nullptr, nullptr);
```

Comment:
If this is really the same as the old stuff, let's not keep the commented code block below, or why would we?